packages/x-charts-pro/src/internals/plugins/useChartProZoom/useChartProZoom.types.ts

```diff
@@ -54,7 +54,16 @@ export interface UseChartProZoomPublicApi {
   setZoomData: (value: ZoomData[] | ((prev: ZoomData[]) => ZoomData[])) => void;
 }
 
-export interface UseChartProZoomInstance extends UseChartProZoomPublicApi {}
+export interface UseChartProZoomInstance extends UseChartProZoomPublicApi {
+  /**
+   * Zoom in the chart.
+   */
+  zoomIn: () => void;
+  /**
+   * Zoom out the chart.
+   */
+  zoomOut: () => void;
```

Comment:
I feel these should be parametrised as:

```suggestion
  /**
   * Zoom in the chart.
   */
  zoomIn: (step: number) => void;
  /**
   * Zoom out the chart.
   */
  zoomOut: (step: number) => void;
```

Reply:
We can, but what's the benefit? It isn't being used. That's why I didn't add it. We can always add it later if needed

Reply:
Nvm, I always get confused between public vs internal api on the plugins 😢